h/streamer/messages.py

```diff
@@ -82,6 +82,14 @@ def handle_message(message, registry, session, topic_handlers):
         handler(message.payload, sockets, registry, session)
 
 
+def handle_user_event(message, sockets, registry, session):
+    for socket in sockets:
+        reply = _generate_user_event(message, socket)
+        if reply is None:
+            continue
+        socket.send_json(reply)
```

Comment:
This function seems to be unchanged, it has just moved in the file

Reply:
Yeah, it's because it was in the middle of the two parts that constitute the annotation message handler. This has probably been messed up by the other PR now.